go.mod

```diff
@@ -1,13 +1,13 @@
 module github.com/kubernetes-csi/lib-volume-populator
 
-go 1.18
+go 1.19
 
 require (
 	github.com/prometheus/client_model v0.3.0
 	github.com/prometheus/common v0.37.0
-	k8s.io/api v0.26.0-rc.0
-	k8s.io/apimachinery v0.26.0-rc.0
-	k8s.io/client-go v0.26.0-rc.0
+	k8s.io/api v0.26.0
+	k8s.io/apimachinery v0.26.0
+	k8s.io/client-go v0.26.0
 	k8s.io/component-base v0.25.4
 	k8s.io/component-helpers v0.25.4
```

Comment:
this should be 0.26 too

Reply:
The script did not update this line somehow, let me manually update it.